scrapyd/website.py

```diff
@@ -140,7 +142,9 @@ def render(self, txrequest):
             for a in ['project', 'spider', 'job']:
                 s += "<td>%s</td>" % getattr(p, a)
             s += "<td></td>"
-            s += "<td>%s</td>" % (p.end_time - p.start_time)
+            s += "<td>%s</td>" % p.start_time.replace(microsecond=0)
+            s += "<td>%s</td>" % p.end_time.replace(microsecond=0)
+            s += "<td>%s</td>" % (p.end_time.replace(microsecond=0) - p.start_time.replace(microsecond=0))
```

Comment:
Well, python just surprised me in a bad way :confused:
We could solve this with a rounding/truncating function
but let's not bother. The current precision suffices.